dev-proxy/UpdateNotification.cs

```diff
@@ -40,12 +40,12 @@ internal static class UpdateNotification
             }
 
             var latestReleaseVersion = latestRelease.Version;
-            var currentVersion = ProxyUtils.ProductVersion;
+            var currentVersion = ProxyUtils.NormalizeVersion(ProxyUtils.ProductVersion);
```

Comment:
We shouldn't normalize the current version, because if you have a beta installed, we want to warn you if there's a stable version available. Let's revert this change.

Reply:
Reverted regression